packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -105,12 +106,19 @@ const _ActionListSection = ({
   testID,
   _hideDivider,
 }: ActionListSectionProps): JSX.Element => {
+  const { theme } = useTheme();
+
   return (
     <BaseBox
       {...makeAccessible({
         role: getActionListSectionRole(),
         label: title,
       })}
+      style={{
+        paddingTop: theme.spacing[3],
+        marginTop: -theme.spacing[3],
+      }}
+      backgroundColor="surface.background.level2.lowContrast"
```

Comment:
This will set static color for actionlist no? ideally it should take from `surfaceLevel` prop. We can set transparent or nothing here to let it read from parent. Currently it breaks when surfaceLevel is set to 3

<img width="672" alt="image" src="https://user-images.githubusercontent.com/30949385/232438054-142d1f06-2a6a-4e84-b1c8-dc3b95a22614.png">

Reply:
Right good catch!

Reply:
Can't add transparent becuase of this https://github.com/razorpay/blade/pull/1095#discussion_r1166693555